main/src/library/List.flix

```diff
@@ -86,6 +86,11 @@ instance Functor[List] {
     def map(f: a -> b & e, xs: List[a]): List[b] & e = List.map(f, xs)
 }
 
+instance Foldable[List] {
+    def foldLeft(f: b -> a -> b & e, s: b, xs: List[a]): b & e = List.foldLeft(f, s, xs)
```

Comment:
I think we should write the signature uncurried as `(b, a) -> b`. Of course they are equivalent, but this is what we do in other please. I know I wrote it wrong in the ticket (copying from the api).

Reply:
This applies everywhere.